docs/changes.md

```diff
@@ -1,6 +1,8 @@
 ## Changes
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
 
+- `12.8` Multiple Updates:
+    - Extend support for doclinks configuration in Manage's server bundle pods ([#594](https://github.com/ibm-mas/ansible-devops/pull/594))
```

Comment:
PR title and entry in the changelog should match.  Can you set this same as the PR title ("Extend support for Manage doclinks configuration") - as it's the slightly shorter of the two :)

Reply:
Done!